_episodes/07-operators.md

```diff
@@ -219,7 +229,7 @@ A filtering condition can be defined by using a Boolean expression described by
 channel
   .of( 1..22, 'X', 'Y' )
   .filter(Number)
-  .filter({it<5})
+  .filter {it<5}
```

Comment:
```suggestion
  .filter { it < 5 }
```
We should make use of white space to encourage readability, and it's use when writing nextflow scripts. 
I'm on the fence about using brackets however. On the one hand, including the brackets enhances clarity of how syntax is parsed, however on the other, leaving them out teaches learners how nextflow is commonly written.

Reply:
About both white-space and brackets: I am not sure which one is more readable, I have seen both in published pipelines. I am OK with both styles however, I think it is not bad to show both usages in training but warn about best practices (e.g. using a persistent style through the whole pipeline).